doc/api/util.markdown

```diff
@@ -335,6 +335,22 @@ util.isError({ name: 'Error', message: 'an error occurred' })
   // false
 ```
 
+Note that this method relies on `Object.prototype.toString()` behavior. It is
+possible to obtain an incorrect result when the `object` argument manipulates
+`@@toStringTag`.
+
+```js
+// This example requires the `--harmony-tostring` flag
```

Comment:
Yes. They are already deprecated in the docs. I think the warning is still worthwhile until they are no longer exposed from core.